spec/views/events/form_examples.rb

```diff
@@ -0,0 +1,52 @@
+require 'rails_helper'
+
+# This set of shared examples provides a configurable way to test event creation and edit forms
+shared_examples 'an event form' do |for_class: Event, path: :new, with: []|
+
+  if path == :new
```

Comment:
This is the same path as in the `elsif` statement. Do you mean `:edit`?

Reply:
yes, thank you ^^"